zebra/zebra_srv6_vty.c

```diff
@@ -271,7 +271,7 @@ DEFUN (no_srv6_locator,
 
 DEFPY (locator_prefix,
        locator_prefix_cmd,
-       "prefix X:X::X:X/M$prefix [func-bits (16-64)$func_bit_len]",
+       "prefix X:X::X:X/M$prefix [func-bits (0-64)$func_bit_len]",
```

Comment:
Please update the documentation as well.

Reply:
Yes, I updated the docs for configuring the SRv6 locator prefix. The description doesn't seem to need to update, so I updated only command usage.